stdlib/public/core/CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ set(SWIFTLIB_ESSENTIAL
   FlatMap.swift
   Flatten.swift
   FloatingPoint.swift
+  FloatingPointRandom.swift
```

Comment:
This can probably be moved down to SWIFTLIB_SOURCES.

Reply:
Done